From 59b43a6583e5cf909192ff8c7314982fc6692298 Mon Sep 17 00:00:00 2001
From: Chris Larson <clarson@kergoth.com>
Date: Mon, 8 Feb 2010 23:38:52 +0000
Subject: [PATCH] Apply patch from solar to also create a tslib.pc.

tslib-$LT_RELEASE.pc will die in a future version, as it's unnecessary.
They'll both stay around for version 1.1 and 1.2.  1.3 or higher will remove
it, unless there's a real need (unlikely).

Signed-off-by: Chris Larson <clarson@kergoth.com>
---
 Makefile.am  |    2 +-
 configure.ac |    1 +
 2 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index a21276b..5383c04 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -16,4 +16,4 @@ ACLOCAL_AMFLAGS = -I m4/internal -I m4/external
 AUTOMAKE_OPTIONS = 1.6
 
 pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = $(PACKAGE)-$(LT_RELEASE).pc
+pkgconfig_DATA = $(PACKAGE)-$(LT_RELEASE).pc $(PACKAGE).pc
diff --git a/configure.ac b/configure.ac
index 6a44ccb..63f5af1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -113,5 +113,6 @@ AC_CONFIG_FILES([Makefile
                  plugins/Makefile
                  src/Makefile
                  tests/Makefile
+                 tslib.pc:tslib.pc.in
                  tslib-$LT_RELEASE.pc:tslib.pc.in])
 AC_OUTPUT
-- 
1.7.8.4

